### PR TITLE
Feature/package test ts

### PR DESCRIPTION
**Pull request by ematipico**
**What kind of change does this PR introduce?**
First typescript test 

**Did you add tests for your changes?**
Yes 

**If relevant, did you update the documentation?**
No

**Summary**
Added first typescript test, really small

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (ev1stensberg) [inline .prettierrc]: we can supply this cli wise

File: packages/info/__tests__/__snapshots__/index.test.ts.snap [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`info should return the information of the enviroment 1`] = `
```
Comment (ev1stensberg) [inline packages/info/__tests__/__snapshots__/index.test.ts.snap]: use regular js for this, no need for snapshot against pure js objects

File: packages/info/index.ts [hunk 3]
```diff
@@ -5,14 +5,18 @@ import * as process from "process";
...
 export default async function info() {
 	process.stdout.write(
-		await envinfo.run({
-			Binaries: ["Node", "Yarn", "npm"],
-			Browsers: ["Chrome", "Firefox", "Safari"],
-			System: ["OS", "CPU"],
-			npmGlobalPackages: ["webpack", "webpack-cli"],
-			npmPackages: "*webpack*",
-		}),
+		await envinfo.run(information()),
```
Comment (ev1stensberg) [inline packages/info/index.ts]: does envinfo write to stdout by default?

File: packages/info/package.json [hunk 4]
```diff
@@ -1,22 +1,40 @@
...
-  },
-  "devDependencies": {
-    "@types/node": "^10.5.1",
-    "typescript": "^2.9.2"
-  },
-  "scripts": {
-    "build": "tsc",
-    "watch": "npm run build && tsc -w"
-  }
+	"name": "@webpack-cli/info",
```
Comment (ev1stensberg) [inline packages/info/package.json]: is this from prettier?

File: packages/info/index.ts (hunk 3 above)
Comment (ematipico) [reply packages/info/index.ts]: I don't know

File: .prettierrc (hunk 1 above)
Comment (ematipico) [reply .prettierrc]: What do you mean?

File: packages/info/package.json (hunk 4 above)
Comment (ev1stensberg) [reply packages/info/package.json]: Ok, discard it

File: packages/info/index.ts (hunk 3 above)
Comment (ev1stensberg) [reply packages/info/index.ts]: Test it locally!

Comment (ematipico) [reply packages/info/index.ts]: Automatically, it doesn't

File: tsconfig.json [hunk 5]
```diff
@@ -7,11 +7,6 @@
...
   },
-  "include": [
-    "packages/**/*.ts"
-  ],
-  "exclude": [
-    "node_modules/**",
-    "packages/*/node_modules/**"
-  ]
+  "include": ["packages/**/*.ts"],
+  "exclude": ["node_modules/**", "packages/*/node_modules/**", "packages/*/__tests__/**"]
```
Comment (ematipico) [inline tsconfig.json]: We don't want to build the files, so for now I added this match to exclude them

Comment (ev1stensberg) [reply tsconfig.json]: Okay!

File: .prettierrc (hunk 1 above)
Comment (ev1stensberg) [reply .prettierrc]: Cli flag for this

**Comment by ev1stensberg** [comment]
If the PR is approved we can do a squash

File: package.json [hunk 6]
```diff
@@ -86,14 +87,7 @@
       "maxSize": "5.32 kB"
     }
   ],
-  "config": {
```
Comment (ematipico) [inline package.json]: Due to archived packaged,  I had to remove this configuration

File: package.json [hunk 7]
```diff
@@ -120,8 +114,6 @@
     "codecov": "^3.1.0",
     "commitizen": "^2.10.1",
     "conventional-changelog-cli": "^2.0.5",
-    "conventional-changelog-lint-config-cz": "^0.3.0",
```
[the remark this answers is not in the document]
Comment (dhruvdutt) [reply package.json]: Why was it archived, any idea? I can't find anything on their readme.

[the remark this answers is not in the document]
Comment (ematipico) [reply package.json]: It now can be done without that package

**Pull request merged**